app/controllers/articles_controller.rb

```diff
@@ -11,13 +10,39 @@ def new
     @article = Article.new
   end
 
+  def edit; end
+
   def create
-    @article = Article.new(params.require(:article).permit(:title, :description))
+    @article = Article.new(article_params)
     if @article.save
       flash[:notice] = 'Article was created successfully.'
       redirect_to @article
     else
       render 'new'
     end
   end
+
+  def update
+    if @article.update(article_params)
+      flash[:notice] = 'Article was updated successfully'
+      redirect_to @article
+    else
+      render 'edit'
+    end
```

Comment:
```suggestion
    unless @article.update(article_params)
      return render 'edit'
    end

    flash[:notice] = 'Article was updated successfully'
    redirect_to @article
```

An alternative way to think about when we try to not nest if-else until we end up with if-else hell